daemon/api.go

```diff
@@ -535,12 +525,7 @@ var getConfigurator = func() configurator {
 
 func snapConfig(c *Command, r *http.Request) Response {
 	vars := muxVars(r)
-	name := vars["name"]
-	developer := vars["developer"]
-	if name == "" || developer == "" {
-		return BadRequest("missing name or developer")
-	}
-	pkgName := name + "." + developer
+	snapName := vars["name"]
```

Comment:
I mentioned it @pedronis as there's a check for name [here](https://github.com/ubuntu-core/snappy/pull/761/files/ede0e845242592e795b273006076f46ac6e3b358#diff-604eb96e84eb2976171c1cedca2bf095R398)